docs/docs/60-new-docs/40-operator-guide/10-basic-installation.md

```diff
@@ -88,6 +94,46 @@ These instructions were tested with:
       --wait
     ```
 
+    :::info
+    Please take a look at the [**Kargo chart documentation**](https://github.com/akuity/kargo/tree/main/charts/kargo) to know all other options that can be set while installing Kargo.
+    :::
+
+### 2. Download Kargo CLI
+
+Go through the [Kargo CLI Installation documentation](../50-user-guide/20-how-to-guides/05-installing-kargo-cli.md) to install the Kargo CLI.
+
+### 3. Access The Kargo API Server
+
+By default, the Kargo API server is not exposed with an external IP. To access the API server, choose one of the following techniques to expose the Kargo API server:
+
+- ### Service Type Load Balancer
+
+Change the kargo-api service type to `LoadBalancer`:
```

Comment:
This is not recommended without being aware of the implications, and suggesting this without either mentioning this or referring to the security document is not advised.

Reply:
I can mention the implications here. Are there any implications specific to Kargo?

Reply:
I do not think we should make any mention of tweaking the Service or Ingress settings in this guide.